EventHubs/tests/test_eventhub_metrics.py

```diff
@@ -30,6 +35,8 @@ def test_pipeline(self):
         self.check_success_log(captured_output)
         self.check_error_log(captured_output)
         self.check_warning_log(captured_output)
+        self.delete_source(collector_id,self.sumo_source_id)
```

Comment:
make these part of teardown override the teardown call the super

Reply:
updated